tests/scratch

```diff
@@ -0,0 +1,148 @@
+Testing FASTA.fna [[3, 1], [4, 1], [5, 1]]
```

Comment:
Unless this file is needed, I'd get rid of it. People should see this when running tests anyway.

Reply:
thanks, deleted